pkgs/development/libraries/libprom/default.nix

```diff
@@ -0,0 +1,39 @@
+{ gccStdenv
+, fetchFromGitHub
+, cmake
+}:
+let
+  stdenv = gccStdenv;
+in
+stdenv.mkDerivation rec {
+  pname = "libprom";
+  version = "0.1.1";
+
+  src = fetchFromGitHub {
+    rev = "v${version}";
+    owner = "digitalocean";
+    repo = "prometheus-client-c";
+    sha256 = "0g69s24xwrv5974acshrhnp6i8rpby8c6bhz15m3d8kpgjw3cm8f";
+  };
+
+  nativeBuildInputs = [ cmake ];
+  doCheck = false;
+
+  patches = [
+    ./prometheus-client-c-0.1.1.patch
+  ];
+
+  preConfigure = ''
+    cd prom
+    export TEST=0
```

Comment:
I know it's totally not your fault, but I *REALLY* hate the necessity of this export... More a question for upstream, but why not default to 0 if it's unset?!

Reply:
Agreed to the hate. I'll probably send them a PR if/when they accept the one I sent already so we can get rid of that patch which is even worse than the export.

Reply:
Could you add a comment why this is needed? Looking at https://github.com/digitalocean/prometheus-client-c/blob/0c15e7e45ad0c3726593591fdd7d8f2fde845fe3/prom/CMakeLists.txt#L124-L126, I would expect this enables the tests but then you are not running them.

Reply:
If you don't have `TEST` set at all, building fails (checked locally when I made this comment).

Reply:
Hmm, perhaps `if ($ENV{TEST})` should be `if ("$ENV{TEST}")`.

Reply:
This was fixed already upstream... I don't love this fetchpatch thing but well, operating under the assumption they'll release a new version soon and we'll be able to get rid of those patches.